de/matrix.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+from factorization import cellLineFactorization
+
+
+def cellLineComparision(cellLine1, cellLine2):
+    w1, eta1, annotation1 = cellLineFactorization(cellLine1)
+    w2, eta2, annotation2 = cellLineFactorization(cellLine2)
+
+    norm1 = np.linalg.norm(w1)
+    norm2 = np.linalg.norm(w2)
+
+    line1_as_set = set(annotation1)
+    intersection = line1_as_set.intersection(annotation2)
+    intersection_annotation = list(intersection)
+
+    index_list1 = []
+    index_list2 = []
+
+    for x in intersection_annotation:
+        index_value1 = annotation1.index(x)
+        index_list1.append(index_value1)
+
+    index_list1.sort()
+    return index_list1, norm1, norm2
+
+
+cellLineComparision('A375', 'HT29')
```

Comment:
You don't want to run functions like this in a file, because they run on *import*. This means that even if I am using the Python package to do something else, these are going to run. However, it *would* be good to make a test that runs these. I'll ask Farnaz to help with that.